step-function/get-files/test_get_files.py

```diff
@@ -85,12 +85,17 @@ def test_get_files(s3_stubber: Stubber):
             'Key': 'myJobId/myBrowse.png',
             'Size': 10,
         },
+        {
+            'Key': 'myJobId/myBrowse_rgb.png',
+            'Size': 10,
+        },
     ]
     stub_list_files(s3_stubber, 'myJobId', 'myBucket', files)
     stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myProduct.zip', 'product')
     stub_expiration(s3_stubber, 'myBucket', 'myJobId/myProduct.zip')
-    stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myThumbnail.png', 'thumbnail')
-    stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myBrowse.png', 'browse')
+    stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myThumbnail.png', 'amp_thumbnail')
+    stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myBrowse.png', 'amp_browse')
+    stub_get_object_tagging(s3_stubber, 'myBucket', 'myJobId/myBrowse_rgb.png', 'rgb_browse')
```

Comment:
consider amp-thumbnail, amp-browse, rgb-browse to be consistent with what's implemented in rtc gamma, but the implementation happily supports either so it's not an actual issue